website/source/docs/builders/azure.html.md

```diff
@@ -91,7 +91,7 @@ you should specify `subscription_id`, `client_id` and one of `client_secret`,
 The Azure builder can create either a VHD, or a managed image. If you are
 creating a VHD, you **must** start with a VHD. Likewise, if you want to create
 a managed image you **must** start with a managed image. When creating a VHD
-the following two options are required.
+the following four options are required.
```

Comment:
I imagine that we're gonna end up in a situation where this changes again; let's just future-proof it by changing it to 

```suggestion
the following options are required.
```